src/containers/AuthenticationRoute/index.js

```diff
@@ -1,26 +1,40 @@
 import React from "react";
 import { Route, Redirect, withRouter } from "react-router-dom";
+import AuthContext from "./../../context/AuthContext";
 
 class AuthenticationRoute extends Route {
+  // Assign a contextType to get isAuthed from AuthContext
+  static contextType = AuthContext;
```

Comment:
Again you shouldn't import context directly

Reply:
Hooks don't work inside classes. Don't know other way.

Reply:
Instead of static contextType, now  I use AuthContext.Consumer here to grab isAuthed